recipes/sage-proteomics/meta.yaml

```diff
@@ -16,12 +16,11 @@ source:
 
 requirements:
   build:
-    - rust >=1.62
+    - {{ compiler('c') }}
   host:
-
+    - rust >=1.18.0
```

Comment:
Sage has a MSRV (minimum supported rust version) of 1.62
It will not compile on any version less than 1.62